src/public/stylesheets/style.css

```diff
@@ -954,11 +989,6 @@ li.dropdown-submenu:hover > ul.dropdown-menu {
     background-color: var(--launcher-pane-background-color);
 }
```

Comment:
Still pertaining to style, something happened with the note preview, showing some strange boxes:

<img width="1023" alt="image" src="https://github.com/user-attachments/assets/1075d500-f04c-41d9-b28f-eaa85e8d0a37">

Reply:
It doesn't show from my side, or I fix it from some changes I don't know. Could you double check?
![image](https://github.com/user-attachments/assets/2ad72e49-56bd-4405-8ead-e74136aaac1f)